### PR TITLE
merged to master

### PR DESCRIPTION
**Pull request by ksrajath7**

### CODE REVIEW

File: types.ts [hunk 1]
```diff
@@ -7,13 +7,13 @@ import { RouteProp } from '@react-navigation/native';
 export interface Entry {
     id: number,
     date: Date,
-    image?: string,
+    image?: Array<string>,
```
Comment (naviji) [inline types.ts:10]: image ella entrykkum undaville - empty array. Appo evide question mark venda.

File: screens/Home.tsx [hunk 2]
```diff
@@ -280,16 +393,99 @@ export default function Home({ navigation }: HomeProps) {
...
+          colors={[backTopColor, backBottomColor]}
+          style={{
+            position: 'absolute',
+            left: 0,
+            right: 0,
+            top: 0,
+            height: height+80,
+          }}
+        /> */}
+      {/* <Image source={require("../assets/ab2.jpg")}  blurRadius={10} style={{flex:1,opacity:0.9, position:"absolute",height:"110%", width:"100%", }} resizeMode="cover"></Image> */}
```
Comment (naviji) [inline screens/Home.tsx:456]: ethu delete cheytho

Comment (ksrajath7) [reply screens/Home.tsx]: athippo thalkalam avde irunnotte.. njan chalapo use cheyyum

File: types.ts (hunk 1 above)
Comment (ksrajath7) [reply types.ts:10]: ok

**Pull request closed without merging**